### ISSUE TITLE
2 Kick 1 buttons

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/center-and-variability#402]
**Test device**
Samsung
**Operating System**
Win 11
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/964
If you have kicked 14 balls onto the field there will be two buttons that say "Kick 1" which looks odd. It may be worth taking a second look at this design.

**Visuals**
![secondkickone](https://github.com/phetsims/center-and-variability/assets/41024075/6889bc26-c87d-48cb-948f-2e89a8215bec)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Center and Variability‬
URL: https://phet-dev.colorado.edu/html/center-and-variability/1.1.0-dev.12/phet/center-and-variability_all_phet.html
Version: 1.1.0-dev.12 2023-07-24 17:55:50 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36
Language: en-US
Window: 1536x707
Pixel Ratio: 1.25/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 4096
Texture: size: 8192 imageUnits: 32 (vertex: 32, combined: 64)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by catherinecarter** [comment]
Good suggestion, Katie. I'll talk to the developers and see if we can make the bottom button invisible when there's only one kick left.

**Comment by catherinecarter** [comment]
@marlitas and @samreid - is this something doable?

**Comment by samreid** [comment]
Yes, it is certainly doable and I believe we have considered it. However, we would need to decide/design how the screen reader/keyboard navigation/focus support would work. In my opinion, it is not pedagogically harmful to show two "kick 1" buttons, so we should consider the complications from a11y as part of this design. I'll reassign @catherinecarter for a quick check-in.

**Comment by samreid** [comment]
@marlitas indicated that we have a similar problem when both buttons disappear (focus would disappear from them). So it seems we should remove the excess button.

@samreid asks: if focus is on the lower kick 1 button, should we instead hide the upper kick 1 button?

We also discussed that it could be confusing if the user wants to press the lower button over and over, and it disappears when there are still soccer balls left to kick.

We also discussed that the lower button "tells you" how many soccer balls are left.

In summary,
@catherinecarter doesn't feel strongly one way or the other
@marlitas thinks it is odd to show two identical buttons at the same time
@samreid thinks hiding it will cause more problems than it solves (not feeling too strongly, like 60/40). It's a bad user experience to hide a button right before the user clicks on it. Or to move focus away from a button that the user has focused.
@KatieWoe said it did seem odd, but not problematic. Probably wouldn't trip anyone up. This is more of a nit-picky problem. Feel free to ignore if that's the best option.
@amanda-phet in a prior thread said the double identical buttons were OK, if others were OK with it.

@catherinecarter suggested implementing a rough draft of it, to see how we feel?
